FLEDGE.md

```diff
@@ -676,14 +699,17 @@ The arguments to this function are:
       'componentSeller': 'https://www.some-other-ssp.com',
       'interestGroupOwner': 'https://www.example-dsp.com/',
       'renderURL': 'https://cdn.com/url-of-winning-creative.wbn',
-      'bid:' bidValue,
+      'bid': bidValue,
+      'bidCurrency': ...,
       'desirability': desirabilityScoreForWinningAd,
       'topLevelSellerSignals': outputOfTopLevelSellersReportResult,
       'dataVersion': versionFromKeyValueResponse,
       'modifiedBid': modifiedBidValue,
-      'highestScoringOtherBid': highestScoringOtherBidValue
+      'highestScoringOtherBid': highestScoringOtherBidValue,
+      'highestScoringOtherBidCurrency': ...
     }
     ```
+    * `bidCurrency` and `highestScoringOtherBidCurrency` provide (highly redacted) information on what currency the corresponding numbers are in. Please refer to the section on [Reporting in Multi-Currency Auctions](#53-reporting-in-multi-currency-auctions) for more details.
```

Comment:
What does "highly redacted" mean here?  I feel like the information is fairly explicit, verified and only leaks contextual information.

Reply:
I don't understand.  If we don't actually specify the concrete buyer currency, what do we specify?  This also seems to contradict the "information on the currency of the corresponding numbers".

Reply:
Well, that's what section 5.3 talks about...?

Reply:
"highly redacted" sounds like it's always redacted.  Can we reword this?  I'd suggest one of:

1. instead of "(highly redacted)", saying ", potentially redacted (e.g. redacted if [explain when redaction could happen])", or
2. if there is a simple way to codify the logic then saying something simple like "bidCurrency is in the buyer's perBuyerCurrency if that matches generateBid()'s bidCurrency, otherwise sellerCurrency"

Reply:
Again, if explaining the rules here were possible, I wouldn't need an entire session about it. Maybe I should actually say less?

(Also PTAL the other changes)